airgun/entities/host.py

```diff
@@ -46,13 +46,19 @@ def create(self, values):
         host_view.flash.assert_no_error()
         host_view.flash.dismiss()
 
-    def get_register_command(self, values, full_read=None):
+    def get_register_command(self, values=None, full_read=None):
         """Get curl command generated on Register Host page"""
         view = self.navigate_to(self, 'Register')
-        view.fill(values)
-        self.browser.click(view.generate_command)
-        self.browser.plugin.ensure_page_safe()
-        view.registration_command.wait_displayed()
+        if values is not None:
```

Comment:
```suggestion
        if not values
```

Reply:
`not None => True` will be wrong condition here, we need to fill when there are values

Reply:
I mean `if values`